versions/changelogs/current.md

```diff
@@ -1,6 +1,7 @@
 # Changes to the oeplatform code
 
 ## Changes
+- All visitors can use metadata builder in standalone version
```

Comment:
We have this new convention that we also link to the PR in the changelog. This helps to find the changes that are related to the changelog entry.

```suggestion
- All visitors can use metadata builder in standalone version  [(#1746)](https://github.com/OpenEnergyPlatform/oeplatform/pull/1746)
```